deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/indexer-0-identity.encrypted

```diff
@@ -0,0 +1,22 @@
+{
+	"data": "ENC[AES256_GCM,data:jb2V1wNzhS3AnYnLnePaSI7ppLmsCK3fD1sET/8qv3aibX5F7pG0Nyd/8Hy4CfHy30Y7UCUBXnhCAF0AvTwMFnCjVMY=,iv:w8Inv/LhSoL3/3QVmLhVzv2kb57AXP9ZJ//V30K2P0Y=,tag:kc0YFmpPnY2uyavnbzdneg==,type:str]",
+	"sops": {
+		"kms": [
+			{
+				"arn": "arn:aws:kms:us-east-2:407967248065:alias/sti_flux",
+				"created_at": "2022-03-30T15:24:20Z",
+				"enc": "AQICAHi99m+hXhXmjeAqO3v9MBDcMaC2zWJIlamGyiBfOVdULAGo3ZZUKyGQs1qIMKXJEsTPAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMHytgB5OTIK0A7oPfAgEQgDv62bWAFPeEOQoiy/CL17ebjkW0WH53kV0hLxtzPe/W6H93WCSoZyYt5dHvEWvyaCk5IjoJs3ghXggziw==",
+				"aws_profile": ""
+			}
+		],
+		"gcp_kms": null,
+		"azure_kv": null,
+		"hc_vault": null,
+		"age": null,
+		"lastmodified": "2022-03-30T15:24:21Z",
+		"mac": "ENC[AES256_GCM,data:kQUVcjzcGdnizPGPSZzkcTHZuLzDX2ad4+0xponQE447u64i+BGel03PoqMGEBVQ3baEhVJJGshG1TumQO8xP/bMdSw49/K/0epFWXcIllq+xdTACj5b7lBZOLWtbTcWgE9VsvqP+F+VBPNrME3k0RfO4HrWVLRwB7KnJECXOus=,iv:N/2xwM6n1ogP7Y3AOzQUYl24+2F0KCPEbL8nzwCankw=,tag:GOD0RflvQJuWcbiDbi9ALQ==,type:str]",
+		"pgp": null,
+		"encrypted_regex": "^(data|stringData)$",
+		"version": "3.7.2"
+	}
+}
```

Comment:
Please rename this file to say `identity.key.encrypted`? `indexer-0` no longer means anything.